src/engines/kotor/gui/chargen/chargenname.cpp

```diff
@@ -36,23 +39,41 @@ CharacterGenerationNameMenu::CharacterGenerationNameMenu(CharacterGenerationInfo
 
 	load("name");
 
+	_name = "";
```

Comment:
That isn't necessary. A Common::UString is initialized with an empty string by default.